odbc/1.introduction.md

```diff
@@ -0,0 +1,23 @@
+# ODBC
+
+Open Database Connectivity (ODBC) is a standardized API for connecting to databases that is cross-platform and cross-DBMS. By installing an general ODBC driver manager and a driver for your specific DBMS, the code that interacts with the ODBC API can be generalized. This makes ODBC a very powerful tool for both development and for production, as the application can be deployed on any system and talk to any DBMS as long as it has the correct drivers installed.
+
+## Why ODBC?
+
+For the open-source software team, ODBC is the preferred method of connecting to Db2 the IBM i.
+
+* Because ODBC is cross-platofrm, it allows you to develop your applications directly on your development machine, whether its a Linux, Windows, or directly on the IBM i. Database connectors built for the IBM i specifically often cannot be built or run on a non-IBM i system.
+
+* When it is time to deploy your application on your IBM i, you can simply move your code from your development machine to your IBM i. Your database commands will remain unchanged, and as long as you have ODBC set up on your IBM i as well, everything will _just work_.
```

Comment:
"your IBM i" -> "your IBM i system"

Reply:
Also have the comma in the wrong place:

"unchanged, and as long" -> "unchanged and, as long"